src/core/operations_problem.jl

```diff
@@ -429,15 +438,17 @@ function build_pre_step!(problem::OperationsProblem)
             reset!(problem)
         end
         settings = get_settings(problem)
+        if !get_use_parameters(settings)
+            set_use_parameters!(settings, built_for_simulation(problem))
+            @debug "Set use_parameters = true for use in simulation"
```

Comment:
Tag with a `_group`?

Reply:
I think we should address the groupings of the @debug in a separate PR to implement it for all.